lib/fastlane/plugin/wpmreleasetoolkit/helper/git_helper.rb

```diff
@@ -124,7 +124,18 @@ def self.list_local_tags(matching: '*')
       # @param [Bool] delete_on_remote If true, will also delete the tag from the remote. Otherwise, it will only be deleted locally.
       #
       def self.delete_tags(tag_names, delete_on_remote: false)
-        Action.sh('git', 'tag', '-d', *tag_names)
+
+        tag_names = [tag_names] unless tag_names.is_a? Array
+
+        g = Git.open(Dir.pwd)
+        local_tag_names = g.tags.map do |tag|
+          tag.name
+        end
+
+        tag_names.each do |tag|
+          Action.sh('git', 'tag', '-d', *tag) if local_tag_names.include? tag
+        end
+
         if delete_on_remote
           remote_refs = tag_names.map { |tag| ":refs/tags/#{tag}" }
           Action.sh('git', 'push', 'origin', *remote_refs)
```

Comment:
Ditto, since you already use the `Git` gem, that would be a good occasion to migrate this call to the `Git` gem's method equivalent that would make it nicer.